source/features/global-search-filters.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {XIcon} from '@primer/octicons-react';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import SearchQuery from '../github-helpers/search-query';
+
+const filters = [
+	['Forks', 'fork:true'],
+	['Private', 'is:private'],
+	['Yours', 'user:@me']
+];
+
+function init(): void {
+	const items = [];
+	for (const [name, filter] of filters) {
+		const item = <a className="filter-item" href={location.href}>{name}</a>;
+		const query = new SearchQuery(item as unknown as HTMLAnchorElement);
+
+		if (query.includes(filter)) {
+			query.remove(filter);
+			item.classList.add('selected');
+			item.prepend(<XIcon className="float-right"/>);
+		} else {
+			query.add(filter);
+		}
+
+		items.push(<li>{item}</li>);
+	}
+
+	select('#js-pjax-container .menu')!.nextElementSibling!.after(
```

Comment:
This adds it in place of the languages; it should be added below it to avoid moving the UI too much. Also it's best to use a better selector instead of `nextElementSibling`